ts/client/config.ts

```diff
@@ -1,12 +1,13 @@
+import { BigNumberish } from "ethers";
 import { readFile } from "fs";
 import { resolve } from "path";
 import { promisify } from "util";
 
 const readFileAsync = promisify(readFile);
 
 export type FeeReceivers = Array<{
```

Comment:
What do you think we make FeeReceivers a class, so we can build the get random token getter or iterators from the object method?

Reply:
We can do that, I see the random part as more test related but no harm in having that available in prod code.

Reply:
https://github.com/thehubbleproject/hubble-contracts/issues/628